docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
Per our F2F discussion, I agree that unifying image and root file system thresholds makes sense given that we choose to handle the threshold the same for both scenarios.